CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 - Improve our mixin structure (breaking change)
 - Adds HasGameRef mixin
 - Fixes for ComposedComponent
+- Added no-parameter alias functions for setting the game's orientation.
+- Updated documentation on Util section.
```

Comment:
Not necessary to add entries for doc updates

Reply:
removed